dotnet/content/install.ps1

```diff
@@ -1,22 +1,46 @@
- param (
-    [Parameter(Mandatory=$true)][string]$ExtensionVersion
-)
-
 # https://github.com/projectkudu/kudu/blob/13824205c60a4bdb53896b9553ef1f370a93b911/Kudu.Services/SiteExtensions/SiteExtensionController.cs#L240
 
+$Version="vUNKNOWN"
+
+function Log ([string] $text)
+{
+    $LogDate=Get-Date -Format "MM/dd/yyyy HH:mm K"
```

Comment:
Just a suggestion about format, I think "sortable" suits better for logs?
```suggestion
    $LogDate=Get-Date -Format "yyyy-MM-dd HH:mm K"
```

Reply:
Done